pkg/auth/webapp/csrf_middleware.go

```diff
@@ -82,11 +82,19 @@ func (m *CSRFMiddleware) unauthorizedHandler(w http.ResponseWriter, r *http.Requ
 	strictCookie, err := m.Cookies.GetCookie(r, CSRFDebugCookieSameSiteStrictDef)
 	hasStrictCookie := (err == nil && strictCookie.Value == "exists")
 
+	csrfCookie, _ := r.Cookie(m.CookieDef.Name)
+	csrfCookieSizeInBytes := 0
+	if csrfCookie != nil {
+		// do not return value but length only for debug.
+		csrfCookieSizeInBytes = len([]byte(csrfCookie.Value))
+	}
```

Comment:
The format is documented here https://github.com/gorilla/securecookie/blob/main/securecookie.go#L281 Try to mask the value and the sig, and include it as well.

Reply:
https://github.com/gorilla/securecookie/blob/eae3c1840ec4adda88a4af683ad0f60bb690e7c2/securecookie.go#L320C30-L320C44

seems it strips the name before sending and add it back only during decode, so the payload format is `date|value|mac`

let's mask the value and mac here?

Reply:
![CleanShot 2024-06-19 at 21 46 52@2x](https://github.com/authgear/authgear-server/assets/11769136/90497138-1254-4055-9aa3-fb9ce29064af)

Reply:
OK.